### ISSUE TITLE
ExtMod?

### ISSUE THREAD

**Issue by nschneid** [nert-nlp/cgel#45]
As @BrettRey and I discussed today, PreDetMod seems too specific as a function, but it would be nice to distinguish all external modifiers. How easy would it be to find them all?

**Comment by BrettRey** [comment]
External modifiers in NPs should have an NP as a sibling.

The rest is spitballing:
By analogy, any mod with a clause as a sibling might be an external modifier, so any pre-marker modifier in a clause. _Not_, _even_, _just_, _also_, and a few other adverbs might commonly appear as peripheral modifiers in other phrases (e.g., _even after eliminating outliers_).

**Comment by nschneid** [comment]
We have a case of "literally" modifying an NP

**Comment by BrettRey** [comment]
I think that's a good example of an external modifier.

**Comment by nschneid** [comment]
Chris Traeger would agree.

**Comment by BrettRey** [comment]
literally

**Comment by nschneid** [comment]
I see you put Mod_ext in the doc. That works for me.

**Comment by nschneid** [comment]
Looking at CGEL, it seems the use of the term "external modifier" is limited to NP structure. Is it equivalent to saying a Mod outside the Nom? If so I suppose we don't need to designate a separate function.

Focusing modifiers are not limited to NPs, but I'm not sure we want to broaden the term "external modifier" to include them when they modify VPs, PPs, etc.

**Comment by BrettRey** [comment]
Yes, only in NPs in CGEL, but I think it would be useful to say that _even_ in _Even that he could is in question_ is an external modifier, or _even after dark_.

**Comment by nschneid** [comment]
Here are words marked as adverbial modifiers of non-copular nominals (including PPs) in UD: http://universal.grew.fr/?custom=62fd6a084013d

Several interesting constructions here.

**Comment by BrettRey** [comment]
Can you say more about these?

**Comment by BrettRey** [comment]
Here is a list of "adverbs" that occur at the start of a sentence before a _that_ clause. Some of them are obviously the result of mistagged _that_.

**Comment by BrettRey** [comment]
Here is a list of "adverbs" that occur at the start of a sentence before a _that_ clause. Some of them are obviously the result of mistagged _that_. The only cases that strike me a external modifiers are _just_, _also_ (?), and _even_; to these I would add _simply_. I think _maybe_, _perhaps_, and _somehow_, are not semantically modifying the _that_ clause but a higher VP, and, while _namely_ certainly is, syntactically it has to follow another _that_ clause.

On the other end (post-head), _too_ and _even_ again, maybe _anyway_, _also_,  and _indeed_.

**Comment by nschneid** [comment]
And a similar list for UD: http://universal.grew.fr/?custom=62fe4681ea5a8

**Comment by BrettRey** [comment]
So, what's your feeling about external modifiers for non-NPs?

**Comment by nschneid** [comment]
I think it's worth exploring as an enhancement, but not as high-priority as other tasks.

**Comment by BrettRey** [comment]
Fair enough!
